cmd/deployApplication.go

```diff
@@ -28,7 +28,7 @@ const deployApplicationEnabled = true
 var deployApplicationCmd = &cobra.Command{
 	Use:   "deploy",
 	Short: "Will trigger deploy of a Radix application",
-	Long:  `Triggers deploy of a Radix application according to the radix config in its repository's master branch.`,
+	Long:  `Triggers deploy of a Radix application according to the radix config in its repository's main branch.`,
```

Comment:
Chaning default to "main" is a breaking change. I think we should keep the old default

Reply:
Reverted